File: Java/src/Iniciante/Solucao_1010.java

import java.io.IOException;
import java.util.Scanner;

public class Solucao_1010 {

    public static double CalcularValorAPagar(int numerosDePecas, double valorPecas){
        return numerosDePecas * valorPecas;
    }

    public static void main(String[] args)throws IOException{
        var input = new Scanner(System.in);

        int numPecas1, numDePecas1, numPecas2, numDePecas2;
        double valorPeca1, valorPeca2, resultado;

        numPecas1 = input.nextInt();
        numDePecas1 = input.nextInt();
        valorPeca1 = input.nextDouble();

        numPecas2 = input.nextInt();
        numDePecas2 = input.nextInt();
        valorPeca2 = input.nextDouble();

        resultado = CalcularValorAPagar(numDePecas1,valorPeca1) + CalcularValorAPagar(numDePecas2,valorPeca2);

        System.out.printf("VALOR A PAGAR: R$ %.2f%n", resultado);
        input.close();

    }
}

File: Python/Iniciante/Solucao_1010.py
def valor_a_pagar(quantidade_pecas, valor_peca):
    return quantidade_pecas * valor_peca
try:
    num_peca1, quantidade_peca1, valor_peca1 = map(float, input().split())
    num_peca2, quantidade_peca2, valor_peca2 = map(float, input().split())

    num_peca1 = int(num_peca1)
    num_peca2 = int(num_peca2)
    quantidade_peca1 = int(quantidade_peca1)
    quantidade_peca2 = int(quantidade_peca2)

    print(f"VALOR A PAGAR: R$ {valor_a_pagar(quantidade_peca1,valor_peca1) + 
             valor_a_pagar(quantidade_peca2,valor_peca2):.2f}")
except (ValueError, TypeError):
    exit()